.github/dependabot.yml

```diff
@@ -54,6 +54,15 @@ updates:
           - 'puppeteer'
           - 'puppeteer-*'
 
+      types:
+        patterns:
+          - '@types/*'
```

Comment:
@owenatgov I've added a commit to make sure `@types/*` updates like this one are grouped in future as per:

* https://github.com/alphagov/govuk-frontend/pull/3883